docs/vs-2015/snippets/cpp/VS_Snippets_Misc/NVC_MFC_Explorer/cpp/ExplorerView.cpp

```diff
@@ -123,14 +123,14 @@ void CExplorerView::OnSize(UINT nType, int cx, int cy)
 	m_wndList.SetWindowPos (NULL, -1, -1, cx, cy, SWP_NOMOVE | SWP_NOZORDER | SWP_NOACTIVATE);
 }
 
-void CExplorerView::OnActivateView(BOOL bActivate, CView* pActivateView, CView* pDeactiveView) 
+void CExplorerView::OnActivateView(BOOL bActivate, CView* pActivateView, CView* pDeactivateView) 
```

Comment:
Here it is "Activate", so "Deactiveate" would be the appropriate one

Reply:
Or the previous parameter is supposed to be pActiveView.

Reply:
By convention we use the same names in the documentation as in the headers, which are here in the default install location: C:\Program Files (x86)\Microsoft Visual Studio\Preview\Enterprise\VC\Tools\MSVC\14.16.27002\atlmfc\include. Since the headers use pDeactiveView we should not change the doc.

Reply:
@mikeblome Should the middle parameter by pActivateView or pActiveView according to the headers? Which file do I look in under C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\atlmfc\include?